src/components/AdminPage/ArticlesTable.js

```diff
@@ -0,0 +1,83 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Segment, Header, Table, Button, Icon } from 'semantic-ui-react';
+import { getArticles } from '../../actions/articles';
+
+class ArticlesTable extends PureComponent {
+  constructor(props) {
+    super(props);
+    this.renderRows = this.renderRows.bind(this);
+  }
+
+  componentDidMount() {
+    this.props.getArticles();
+  }
+
+  renderRows() {
```

Comment:
Would it be better to move rendering the rows in a different file and returning the rendered rows from there?

Reply:
It would be. It's good practice to split components into smaller ones. A good rule is any `render*()` method should be its own component.

I didn't feel like it though, but I'll make the `Row` into its own component in sha Allah.